pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -438,7 +438,21 @@ func (nc *Controller) doNoScheduleTaintingPass(node *v1.Node) error {
 			}
 		}
 	}
+	if node.Spec.Unschedulable {
+		// If unscheduable, append related taint.
```

Comment:
s/unscheduable/unschedulable

here and the rest of the file

Reply:
ops, let me grep them :)